bidwire/notifiers/bid_notifier.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import os
+import sendgrid
+import bidwire_settings
+from .base_notifier import BaseNotifier
+from sendgrid.helpers.mail import *
+from yattag import Doc
+
+log = logging.getLogger(__name__)
+
+ITEMS_DELIMITER = " ### "
+
+
+class BidNotifier(BaseNotifier):
```

Comment:
OK, let me try to refactor this - thought it might be worthwhile to look at this from an OO perspective, but it makes sense to keep this simple for now.